pmdabpf: fix qa/1810 (oomkill) to run in limited memory footprint

jasonk000 opened the pull request:
resolves #1644

andreasgerstmayr commented:
Thanks for the PR!
When I try it as-is on F36 I'm seeing this error from systemd in 1810.full:
```
Failed to start transient service unit: Connection reset by peer
```

Replacing `>> $here/$seq.full` with `> $here/$seq.full` works though. I'm not sure why that is, afaics appending should work without issues even if the file doesn't exist. Any ideas?

In this case the file doesn't exist, so changing it to `>` shouldn't make any difference to the test.

jasonk000 commented:
> Replacing >> $here/$seq.full with > $here/$seq.full works though. I'm not sure why that is, afaics appending should work without issues even if the file doesn't exist. Any ideas?

The `>>` should be handled by whatever is running the script, which is `#!/bin/sh`. 

Regardless - I've rebased on main and changed to `>`.

andreasgerstmayr commented:
I was also quite surprised why `>>` vs `>` makes any difference regarding the systemd call.
Yep, I also think it has to do with the stdout file descriptor.
